packages/react-native-media-player/ios/MediaPlayer.swift

```diff
@@ -0,0 +1,25 @@
+@objc(MediaPlayer)
+public class MediaPlayer: NSObject {
+
+  @objc(load:resolver:rejecter:)
+  public func load(to url: String, resolve: RCTPromiseResolveBlock, reject: RCTPromiseRejectBlock) {
+    // Date is ready to use!
+    resolve(NSNull());
+  }
+  
+  @objc(pause)
+  public func pause() {
+    print("pause")
+  }
+
+  @objc(play)
+  public func play() {
+    print("play")
+  }
+
+  @objc(init)
+  public func init() {
+    print("init")
+  }
+  
+}
```

Comment:
Codacy found an issue: [Files should have a single trailing newline.](https://app.codacy.com/manual/yajananrao/Nadha/pullRequest?prid=5728824)